src/main/kotlin/org/rust/lang/core/resolve/Processors.kt

```diff
@@ -32,10 +32,20 @@ interface ScopeEntry {
  * to the resolve processor
  */
 enum class ScopeEvent : ScopeEntry {
-    // Communicate to the resolve processor that we are about
-    // to process wildecard imports. This is basically a hack
-    // to make winapi 0.2 work in a reasonable amount of time.
-    STAR_IMPORTS;
+    /**
+     * Communicate to the resolve processor that we are about to process wildecard imports.
+     * This is basically a hack to make winapi 0.2 work in a reasonable amount of time.
+     */
+    STAR_IMPORTS,
+    /**
+     * Communicate to the resolve processor that we are about to process dependency crates
+     * (regardless of `extern crate` declarations). It's used to support 2018 edition
+     * (and Rust 1.30) paths, where non-global path can start with a crate name, but
```

Comment:
`since Rust 1.30`

Reply:
I mean that we should slightly rewrite this part of the comment to clarify that such resolve should work in edition 2018 and in 2015 edition since rust 1.30